AmplifyPlugins/API/AWSAPICategoryPlugin/Interceptor/RequestInterceptor/UserPoolRequestInterceptor.swift

```diff
@@ -24,13 +24,13 @@ struct UserPoolURLRequestInterceptor: URLRequestInterceptor {
         guard let mutableRequest = (request as NSURLRequest).mutableCopy() as? NSMutableURLRequest else {
             throw APIError.unknown("Could not get mutable request", "")
         }
-
         mutableRequest.setValue(NSDate().aws_stringValue(AWSDateISO8601DateFormat2),
-                                forHTTPHeaderField: URLRequestContants.Header.xAmzDate)
-        mutableRequest.setValue(URLRequestContants.ContentType.applicationJson,
-                                forHTTPHeaderField: URLRequestContants.Header.contentType)
-        mutableRequest.setValue(URLRequestContants.UserAgent.amplify,
-                                forHTTPHeaderField: URLRequestContants.Header.userAgent)
+                                forHTTPHeaderField: URLRequestConstants.Header.xAmzDate)
+        mutableRequest.setValue(URLRequestConstants.ContentType.applicationJson,
+                                forHTTPHeaderField: URLRequestConstants.Header.contentType)
+        let serviceConfiguration = AmplifyAWSServiceConfiguration(region: .Unknown, credentialsProvider: nil)
+        mutableRequest.setValue(serviceConfiguration.userAgent,
+                                forHTTPHeaderField: URLRequestConstants.Header.userAgent)
```

Comment:
This seems a bit odd to me that we have to instantiate something to get a static value, can we just do 
`AmplifyAWSServiceConfiguration.baseUserAgent`?

Reply:
Hmm that makes way more sense... thanks updating.